README.md

```diff
@@ -112,15 +112,27 @@ To illustrate, `bin/rails console` will run the console in the docker container
 
 https://railsapibasers.docs.apiary.io/
 
-With [RSpec API Doc Generator](https://github.com/zipmark/rspec_api_documentation) you can generate the docs after writing the acceptance specs.
+With [RSpec API Doc Generator](https://github.com/exoego/rspec-openapi) you can generate the docs after writing requests specs.
+### Updating docs
```

Comment:
should we move this section into `docs/api_documentation.md` and expand a bit more what is the process we want to follow (what and when are things ran, locally, in the PR CI, and in the trunk (main) CI)

Reply:
Maybe it's worth moving it to a separate file, but most of the CI comments are addressed in the `ci.md` file.

Reply:
but autogenerating docs isn't part of the CI, it's a process on itself, and we use the CI just to check that they get updated

Reply:
conviced. Will move